configs/exp.yaml

```diff
@@ -0,0 +1,77 @@
+modules:
+  - seacliff
+  - galsim_extra
+
+image:
+  type: RubinCalexp
```

Comment:
Maybe mark the aspirational types as #TODO to remind us what's still left to be implemented.